lib/cli/interactive-setup/console-dev-mode-feed.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const { writeText, style } = require('@serverless/utils/log');
+const WebSocket = require('ws');
+const chalk = require('chalk');
+const { devModeFeed } = require('@serverless/utils/lib/auth/urls');
+const consoleUi = require('@serverless/utils/console-ui');
+const streamBuffers = require('stream-buffers');
+const apiRequest = require('@serverless/utils/api-request');
+const promptWithHistory = require('@serverless/utils/inquirer/prompt-with-history');
+
+const streamBuff = new streamBuffers.ReadableStreamBuffer({
+  frequency: 500,
+  chunkSize: 2048 * 1000000,
+});
+
+const consoleMonitoringCounter = {
+  logBatches: 0,
+  events: 0,
+  responses: 0,
+};
+
+const handleSocketMessage = (data) => {
+  try {
+    const splitData = data.toString('utf-8').split(';;;');
+    const jsonArray = splitData
+      .filter((item) => item !== '' && Array.isArray(JSON.parse(item)))
```

Comment:
Wouldn't `item.startsWith('[') be good enough here? (relying on `JSON.parse` in filter and then repeating it in map doesn't feel optimal)

Reply:
This is a good idea, thanks for the suggestion 😎